PopupTabLayout/PopupTabLayout.js

```diff
@@ -392,12 +408,10 @@ const tabPanelsHandlers = {
  * @extends sandstone/Panels.Panels
```

Comment:
Le'ts add @mixes i18n/I18nDecorator.I18nContextDecorator

Reply:
It's added.